src/components/views/game/gamesummary/GameSummary.tsx

```diff
@@ -2,14 +2,15 @@ import React from "react"
 import { observer } from "mobx-react-lite"
```

Comment:
The view looks nice
![grafik](https://user-images.githubusercontent.com/1506818/170439665-1f857246-5b49-46f1-851b-05bbb6c7a67b.png)


I would put the number inside the li somehow, and align the player name left

Reply:
I am still trying to understand the exact workings as it looks differnt in different browsers (Chrome vs Firefox)